decompiler/config/jak2/all-types.gc

```diff
@@ -25472,37 +25618,35 @@
    (backup-nav-radius       float                    :offset-assert 44)
    (cam-side-shift          float                    :offset-assert 48)
    (enable-cam-side-shift   basic                    :offset-assert 52)
-   (gun?                    basic                    :offset-assert 56)
-   (controls                vehicle-controls :inline :offset-assert 60)
-   (accel-array             vector           :inline :offset-assert 80)
-   (local-accel             vector           :inline :offset-assert 208)
-   (pilot-trans             vector           :inline :offset-assert 224)
-   (pilot-quat              vector           :inline :offset-assert 240)
-   (pilot-scale             vector           :inline :offset-assert 256)
-   (pilot-time              time-frame                   :offset-assert 272)
-   (as-daxter?              basic                    :offset-assert 280)
-   (art-group-backup        basic                    :offset-assert 284)
+   (gun?                    symbol                   :offset-assert 56)
+   (controls                vehicle-controls         :offset 60)
+   (accel-array             vector           :inline :offset 80)
```

Comment:
This could be
```
(accel-array             vector 8         :inline :offset-assert 80)
```
I think. Then this whole type can have `:offset-assert`s too